pyproject.toml

```diff
@@ -27,9 +27,9 @@ requires-python = ">= 3.8, < 4"
 dependencies = [
     "frequenz-api-microgrid >= 0.11.0, < 0.12.0",
     "frequenz-channels >= 0.12.0, < 0.13.0",
-    "google-api-python-client >= 2.71, < 3",
-    "grpcio >= 1.51.1, < 2",
-    "grpcio-tools >= 1.51.1, < 2",
+    "google-api-python-client >= 2.71, < 2.73",
+    "grpcio == 1.51.1",
+    "grpcio-tools == 1.51.1",
```

Comment:
In the last commit freezing more versions you don't mention why it is needed. Also, does it make sense to open issues about the errors you found without limiting the versions like this?